scripts/build-docs.sh

```diff
@@ -1,13 +1,13 @@
 #!/usr/bin/env bash
 
 for EXTENSION in cpp hpp ipp; do
-    find . -name "*.${EXTENSION}" -exec /doxygen_prep.sh {} \;
+  find . -name "*.${EXTENSION}" -exec scripts/doxygen_prep.sh {} \;
```

Comment:
@jwallwork23 I can see how I have caused confusion, but this script is used by the Docker container where doxygen_prep.sh has already been copied to the root directory. I should either change the Dockerfile to account for this, or we should leave this as it was. What do you think is best?

```suggestion
  find . -name "*.${EXTENSION}" -exec /doxygen_prep.sh {} \;
```

Reply:
Ah okay I see. I should've modified the `build_docs` workflow to use the same approach as you did in the Dockerfile.

Reply:
I think I fixed it in https://github.com/nextsimhub/nextsimdg/pull/961/commits/7351464f6164d90a87beb7bf67fb1bef7d31e5ee?

Reply:
Yep, that looks good to me :ok_hand: